### PR TITLE
BI-6469: Added feedback link to confirmation page

### PR DESCRIPTION
**Pull request by faggc**
+ Changed feedback link in banner to new url

## Description

Added feedback link to confirmation page and altered the feedback link in the banner to match the new url

## Jira Ticket

https://companieshouse.atlassian.net/browse/BI-6469

## Coding Standards

Code Style Guide: https://github.com/companieshouse/dissolution-web/wiki/Code-Style-Guide

## Checklist

- [ ] 3 Amigos
- [ ] Acceptance Criteria met
- [ ] Branch named {feature|hotfix|task}/{S4-*}
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] All unit tests passing
- [ ] WAVE accessibility testing tool ran on new or updated pages
- [ ] All UI Tests Passing
- [ ] Pulled latest master into feature branch
- [ ] Sonar Analysis
- [ ] Master branch on Rebel1 and CIDev CI/CD pipeline is green
- [ ] Any Docker environment variables added are consistent with those on our upstream environments (Rebel1, CIDev etc)
- [ ] If your pull request depends on any other, please link them in the description

## Screenshots of new or updated views
Screenshot 2020-12-17 at 10 39 14
Screenshot 2020-12-17 at 10 48 31

### CODE REVIEW

File: src/views/view-final-confirmation.njk [hunk 1]
```diff
@@ -34,6 +34,7 @@
       }}
       
       <p class="govuk-body"><a id="certificate-download" href="{{ Paths.CERTIFICATE_DOWNLOAD_URI }}" target="_blank" class="govuk-link">Download the signed copy of the application (opens in a new tab)</a>.</p>
+      <p class="govuk-body govuk-!-margin-top-4">This is a new service. Help us improve it by providing <a href="{{ feedbackLink }}">feedback</a></p>
```
Comment (mateuszc-kainos) [inline src/views/view-final-confirmation.njk:37]: Is there a full stop missing after the link?

File: src/constants/app.const.ts [hunk 2]
```diff
@@ -6,3 +6,4 @@ export const COMPANY_NOT_ACTIVE_ERROR_MSG = 'The company is not currently active
 export const COMPANY_OVERSEAS_ERROR_MSG = 'This is an overseas company, and cannot be closed using this service.'
 export const NO_ACTIVE_DIRECTORS_ERROR_MSG = 'The company has no active directors.'
 export const NO_ACTIVE_MEMBERS_ERROR_MSG = 'The company has no active members.'
+export const FEEDBACK_LINK = 'https://www.research.net/r/closing-a-company-feedback'
```
Comment (mateuszc-kainos) [inline src/constants/app.const.ts:9]: ACs specify two different addresses. We are using one in both places here. Is this correct?

Comment (faggc) [reply src/constants/app.const.ts:9]: Ahh you are right, ill get that fixed

File: src/views/view-final-confirmation.njk [hunk 3]
```diff
@@ -34,6 +34,7 @@
       }}
       
       <p class="govuk-body"><a id="certificate-download" href="{{ Paths.CERTIFICATE_DOWNLOAD_URI }}" target="_blank" class="govuk-link">Download the signed copy of the application (opens in a new tab)</a>.</p>
+      <p class="govuk-body govuk-!-margin-top-4">This is a new service. Help us improve it by providing <a href="{{ confirmationFeedbackLink }}">feedback</a></p>
```
Comment (mateuszc-kainos) [inline src/views/view-final-confirmation.njk:37]: Are we missing the full stop? The ACs have a '.' at the end.

**Review by mateuszc-kainos — APPROVED** [review]

**Pull request merged**